cat/c11.cat

```diff
@@ -20,7 +20,8 @@ let cacq = ACQ | (SC & (R | F)) | ACQ_REL | (F & CON)
 let crel = REL | (SC & (W | F)) | ACQ_REL
 let ccon = R & CON
 
-let fr = rf^-1 ; mo
+// To be compatible with reads from uninit memory, we use the default definition of fr
+//let fr = rf^-1 ; mo
```

Comment:
Rather than editing this file, I think it is better to have two cat files, one called `_orig` similar to what we did for RISCV.
Since we are going to rely on the default definition of fr what is based on co, we might want to avoid to renaming of `sb=po` and `mo=co`.

Also, it might be a good idea to create two new test classes that extend the C11 ones (locks and lfds) by simply overriding the method selecting the cat file. This was we have all tests run with both models and guaranteeing the same expected result.

Reply:
I can create a separate cat file. 
> Since we are going to rely on the default definition of fr what is based on co, we might want to avoid to renaming of sb=po and mo=co.

I can also put an explicit definition of `fr` into the cat file that uses `mo` instead of `co`.

> Also, it might be a good idea to create two new test classes that extend the C11 ones (locks and lfds) by simply overriding the method selecting the cat file

And changing the options I guess? Cause the original C11 one won't work without init events.
There are technically three cases, all of which should produce the same results
- Old model with all init events
- New model with all init events
- New model without all init events

Reply:
I added the extra c11 cat file, created two new tests (for locks and lfds) and enabled CAAT on all C11 tests (they work now due to improvements in cutting).

Reply:
Do you remember if this was the same reason why we do not have caat enabled for litmus test (at least for those checking safety)

Reply:
Well, we have never enabled CAAT for litmus tests I believe (independent of the memory model). Of course, for C11 we couldn't enable it due to the same problems with cutting. Btw. CAAT is slower on the C11 tests than the assume solver!